test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -5262,20 +5270,20 @@ def status(self, obj_type=SERVER, attrib=None, id=None,
                     (isinstance(attrib, str) and len(attrib.split(',')) == 1)):
                 bsl = self.status(
                     JOB, 'Resource_List.select', id=id, extend='t')
-            if self.scheduler is None:
-                self.scheduler = Scheduler(self.hostname)
-            if 'log_filter' in self.scheduler.sched_config:
-                _prev_filter = self.scheduler.sched_config['log_filter']
+            dflt_sched = self.schedulers[self.dflt_sched_name]
+            if dflt_sched is None:
+                dflt_sched = Scheduler(self.hostname)
+            if 'log_filter' in dflt_sched.sched_config:
```

Comment:
This is JOB related code. 
here given a JOB,  we need to get the sched name. This can be done, through QUEUES. Here are the steps that need to be done. 
1) from job fetch queue
2) from queue fetch partition 
3) from partition fetch scheduler object 
4) update the configs  

The Partition code in PBSPro is not yet to be completed. once we have the implementation ready, we can update this PTL code.